pkgs/tools/misc/webcat/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  pname = "webcat";
+  version = "57a65558f0affac0b2f8f4831c52964eb9ad5386";
+
+  src = fetchgit {
+    url = "https://codeberg.org/rumpelsepp/webcat.git";
+    rev = version;
+    sha256 = "15c62sjr15l5hwkvc4xarfn76341wi16pjv9qbr1agaz1vqgr6rd";
+  };
+
+  vendorSha256 = "1apnra58mqrazbq53f0qlqnyyhjdvvdz995yridxva0fxmwpwcjy";
+
+  meta = with lib; {
+    homepage = "https://rumpelsepp.org/blog/ssh-through-websocket/";
+    description = "The lightweight swiss army knife for websockets";
+    licence = lib.gpl3;
```

```suggestion
    licence = gpl3;
```
I believe you also have to specify `gpl3Only` or `gpl3Plus`, I think this program is `gpl3Plus`.

---

Done in c786f97